astropy/modeling/fitting.py

```diff
@@ -1126,22 +1126,21 @@ def objective_function(self, fps, *args):
         fps : list
             parameters returned by the fitter
         args : list
-            [model, [weights], [input coordinates], {context}]
-        context : dict
-            A dict of pre-computed parameters from the __call__method
-
+            [model, [weights], [input coordinates]]
+        fit_param_indices : list, optional
+            The ``fitparam_indices`` as returned by ``model_to_fit_params``.
+            This must be optional as not all fitters will support it.
```

Comment:
The kwarg is ``fitparm_indices`` and the docstring mentions ``fit_param_indices``. I think it should be the latter ideally?

Also ``objective_function`` is technically public API, but ``model_to_fit_params`` is not, so rather than referring to ``model_to_fit_params``, we should probably just say what it actually is (is it a list/tuple of indices of the parameters actually being fit, excluding e.g. fixed/tied parameters?)

I know the rest of the docstring is a bit useless currently but might as well document this new parameter properly :sweat_smile:

Reply:
I went with `fitparam_indices` for consistency with the internals of `model_to_fit_params`, I can change everything if you think it's worth it?

I might have not been super explicit here because I wasn't 100% sure I could explain what it was :laughing:

Reply:
Fixed